src/misc/programs.cpp

```diff
@@ -1145,7 +1146,8 @@ void PROGRAMS_Init(Section* sec)
 	MSG_Add("CONJUNCTION_AND", "and");
 
 	MSG_Add("PROGRAM_CONFIG_NOT_CHANGEABLE",
-	        "Setting '%s' is not changeable at runtime.\n");
+	        "The '%s' setting can't be changed at runtime, but will be written to a\n"
```

Comment:
"written to the"

Reply:
Don't you wanna also mention that these queued changes will take effect when restarting with `CONFIG -r` or via the reset hotkey?

Reply:
I suggest to reformat it to something like:

_The '%s' setting can't be changed at runtime._`[newline]`
_Additional description. Lorem ipsum dolor sit amet, consectetur adipiscing elit._

This way settings with longer names, like `vga_render_per_scanline`, won't cause a "line break" in an unexpected place.

Reply:
Thanks, yes - added.